crates/db_schema/src/schema.rs

```diff
@@ -797,6 +807,7 @@ joinable!(site_language -> language (language_id));
 joinable!(site_language -> site (site_id));
 joinable!(community_language -> language (language_id));
 joinable!(community_language -> community (community_id));
+joinable!(person_follower -> person (follower_id));
```

Comment:
Seems like you should also have `person_follower -> person(person_id)`, but the code sometimes works without these joinables at all.

Reply:
Tried it, but doesnt compile. Diesel doesnt seem to allow multiple foreign keys to the same table at all.